util.go

```diff
@@ -19,6 +19,14 @@ func renameWithFallback(src, dest string) error {
 		return err
 	}
 
+	// Windows cannot use syscall.Rename to rename a directory
```

Comment:
this is from the go 1.8 release notes:
```
On Unix systems, os.Rename will now return an error when used to rename a directory to an existing empty directory. Previously it would fail when renaming to a non-empty directory but succeed when renaming to an empty directory. This makes the behavior on Unix correspond to that of other systems.
```
https://tip.golang.org/doc/go1.8 so it will fail on unix in go 1.8 as well